backend/persistence/models/audit_log.go

```diff
@@ -0,0 +1,46 @@
+package models
+
+import (
+	"github.com/gofrs/uuid"
+	"time"
+)
+
+type AuditLog struct {
+	ID                uuid.UUID    `db:"id" json:"id"`
+	Type              AuditLogType `db:"type" json:"type"`
+	Error             string       `db:"error" json:"error,omitempty"`
```

Comment:
I think we need to change the datatype of nullable string values to `nulls.String` or something, so we are able to store a NULL value instead of an empty string (I didn't tested it).

Reply:
I don't see any update on this one.